### DISCUSSION TITLE
Welcome to pywinauto_recorder Discussions!

### DISCUSSION THREAD

**Discussion by beuaaa** [beuaaa/pywinauto_recorder#18] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by beuaaa** [comment]
I would like to have some feedback about Pywinauto recorder. Don't hesite to test Pywinauto_recorder.exe and fill in the [form ](https://docs.google.com/forms/d/e/1FAIpQLSdvxXJCYfoFUaTVHCDzGxkMbc_8qq68pTb_7hPbaPUDyYlOeQ/viewform) Thank you for your help.

**Reply by vitalydz** [reply]
Hello, Beuaaa! Can you send me a list of Python files for the exe file creation? I must do the same record/replay on a remote PC client.
